spec/requests/api/v1/credit_notes_controller_spec.rb

```diff
@@ -458,6 +458,32 @@
         expect(json[:credit_notes].pluck(:lago_id)).to contain_exactly matching_credit_note.id
       end
     end
+
+    context "with billing entity code filter" do
+      let(:params) { {billing_entity_code: billing_entity.code} }
+      let(:billing_entity) { create(:billing_entity, organization:) }
+      let!(:matching_credit_note) { create(:credit_note, customer:, invoice: create(:invoice, billing_entity:)) }
```

Comment:
we don't use `let!` use `let` and `before` please.

Reply:
and I think, the same is here...
Will update to not use `let!`